.github/renovate.json5

```diff
@@ -6,6 +6,7 @@
   commitMessageSuffix: " in {{packageFile}}",
   dependencyDashboardAutoclose: true,
   automerge: true,
+  baseBranches: ["main", "/^release\-.*/"],
```

Comment:
Looks like this change caused lots of conflict issues?

- https://github.com/runatlantis/atlantis/pull/3990
- https://github.com/runatlantis/atlantis/pull/3991
- https://github.com/runatlantis/atlantis/pull/3992
- https://github.com/runatlantis/atlantis/pull/3994
- #3982